File: app/src/main/java/wyyu/multi/MainActivity.java
package wyyu.multi;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import wyyu.multi.cell_class.ActivityListClass;
import wyyu.multi.cell_integer.ActivityListInteger;
import wyyu.multi.normal.ActivityListNormal;

public class MainActivity extends AppCompatActivity {

    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.click_normal).setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) {
                ActivityListNormal.open(MainActivity.this);
            }
        });

        findViewById(R.id.click_cell_class).setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) {
                ActivityListClass.open(MainActivity.this);
            }
        });

        findViewById(R.id.click_cell_integer).setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) {
                ActivityListInteger.open(MainActivity.this);
            }
        });
    }
}

File: app/src/main/java/com/wyyu/multiktest/MainActivity.kt
package com.wyyu.multiktest

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.wyyu.multiktest.cell_Integer.ActivityListInteger
import com.wyyu.multiktest.cell_class.ActivityListClass
import com.wyyu.multiktest.databinding.ActivityMainBinding
import com.wyyu.multiktest.normal.ActivityListNormal

class MainActivity : AppCompatActivity() {

    private var binding: ActivityMainBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding?.root)

        initActivity()
    }

    private fun initActivity() {
        val clickListener = View.OnClickListener { view ->
            when (view.id) {
                R.id.click_normal -> {
                    ActivityListNormal.open(this@MainActivity)
                }
                R.id.click_cell_class -> {
                    ActivityListClass.open(this@MainActivity)
                }
                R.id.click_cell_integer -> {
                    ActivityListInteger.open(this@MainActivity)
                }
            }
        }
        binding?.clickNormal?.setOnClickListener(clickListener)
        binding?.clickCellClass?.setOnClickListener(clickListener)
        binding?.clickCellInteger?.setOnClickListener(clickListener)
    }
}